Assets/Scripts/OrchestraArmy/Room/Data/RoomPrefabData.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace OrchestraArmy.Room.Data
+{
+    [Serializable]
+    public class RoomPrefabData
+    {
+        
+        /// <summary>
+        /// Objects for the room surroundings.
+        /// </summary>
+        [field: SerializeField]
```

Comment:
[field: *] is alleen nodig voor properties, hoeft niet voor variablen

Overigens heeft dit geen [SerializeField] nodig dacht ik, aangezien het gwn public is